assets/src/tinymce/utils/index.js

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { dispatch, select } from '@wordpress/data';
+/**
+ * Internal dependencies
+ */
+import name from '../store/name';
+import { webStoriesData } from './globals';
+
+/**
+ * Get current view.
+ *
+ * @return {string}
+ */
+export const currentView = () => {
+  return select(name).getCurrentView();
+};
+
+/**
+ * Check if current view is circle view.
+ *
+ * @return {boolean} Flag.
+ */
+export const isCircleView = () => {
+  return 'circles' === currentView();
+};
+
+/**
+ * Update the view wide settings.
+ *
+ * @param root0
+ * @param root0.fieldObj
+ * @param root0.field
+ * @param root0.isReadonly
```

Comment:
Maybe `args` instead of `root0`?

Also some types & descriptions would be good.

Something like this:

```suggestion
 * @param {Object} args Arguments.
 * @param {Object} args.fieldObj Field object.
 * @param {Object} args.field Field.
 * @param {boolean} [args.isReadonly=false] Whether the field is readonly.
```